doodle/ImagingDS/LongStudy.py

```diff
@@ -111,6 +113,11 @@ def activity_in(self, region: str, time_id: int) -> float:
 
         return numpy.sum(self.masks[time_id][region] * self.array_at(time_id=time_id) * self.voxel_volume(time_id=time_id))
 
+    def density_of(self, region: str, time_id: int) -> float:
+        """Returns the mean density of region of interest, in HU"""
+        return numpy.sum(self.masks[time_id][region] * self.array_at(time_id=time_id) * self.voxel_volume(time_id=time_id)) / (numpy.sum(self.masks[time_id][region]) * self.voxel_volume(time_id=time_id) or 1)
```

Comment:
I recommend simplifying this method using `numpy.mean`. In addition, if you want this method to only work for CT (and therefore ensuring that the returned value is in HU), I recommend adding an assertion:

```python
if self.modality != "CT": 
    raise AssertionError("Modality is not CT")
```